colabcode/code.py

```diff
@@ -14,11 +14,17 @@
 
 
 class ColabCode:
-    def __init__(self, port=10000, password=None, mount_drive=False):
+    def __init__(self, port=10000, password=None, mount_drive=False, add_extensions=None):
         self.port = port
         self.password = password
         self._mount = mount_drive
         self._install_code()
+        if add_extensions is not None:
+            global EXTENSIONS
+            if type(add_extensions) == list and type(add_extensions[0]) == str:
```

Comment:
improve this check. this is not a good check.

Reply:
Are you talking about the `[]`  list as input which would throw an exception?

I was a bit conflicted about adding that check - because I was not sure how check-happy I should be, for example `["1"]` should be an error too!